app/src/main/java/org/simple/clinic/deeplink/DeepLinkUpdate.kt

```diff
@@ -18,23 +18,23 @@ class DeepLinkUpdate : Update<DeepLinkModel, DeepLinkEvent, DeepLinkEffect> {
       model: DeepLinkModel,
       event: UserFetched
   ): Next<DeepLinkModel, DeepLinkEffect> {
-    return if (event.user?.loggedInStatus == User.LoggedInStatus.LOGGED_IN) {
-      checkIfMedicalOfficerHasCapabilitiesToTeleconsult(model, event)
+    return if (event.user?.isUserLoggedIn == true) {
```

Comment:
Do we need the `== true` since this returns a boolean?

Reply:
It had an error and was asking for not-null assertion, which wasn't a wise choice considering user can also have a different status. WDYT?

Reply:
Ah, I see. Let's leave it in for now.